src/pycram/process_modules/icub_process_modules.py

```diff
@@ -0,0 +1,687 @@
+import math
+from threading import Lock
+from typing import Union
+from typing_extensions import Any
+import actionlib
+from .default_process_modules import DefaultMoveJoints, DefaultMoveArmJoints, DefaultMoveTCP, DefaultNavigation, \
+    DefaultMoveHead, DefaultDetecting, DefaultMoveGripper
+from .. import world_reasoning as btr
+import numpy as np
+from ..process_module import ProcessModule, ProcessModuleManager
+from ..external_interfaces.ik import request_ik
+from ..ros.logging import logdebug
+from ..utils import _apply_ik
+from ..local_transformer import LocalTransformer
+from ..designators.object_designator import ObjectDesignatorDescription
+from ..designators.motion_designator import MoveMotion, LookingMotion, \
+    DetectingMotion, MoveTCPMotion, MoveArmJointsMotion, WorldStateDetectingMotion, MoveJointsMotion, \
+    MoveGripperMotion, OpeningMotion, ClosingMotion
+from ..robot_description import RobotDescription
+from ..datastructures.world import World
+from ..world_concepts.world_object import Object
+from ..datastructures.pose import Pose
+from ..datastructures.enums import JointType, ObjectType, Arms, ExecutionType, GripperState
+from ..external_interfaces import giskard
+from ..external_interfaces.robokudo import *
+import yarp
+
+from pycram.yarp_utils.yarp_networking import NO_ACK_VOCAB, ACK_VOCAB, open_rpc_client_port, open_buffered_bottle_port, \
```

Comment:
I think it is better to move the calls to yarp into its own file and put this in external_interfaces instead of yarp_utils. So people know what purpose yarp has.

Reply:
Done